frontend/src/api/actions/applicants.js

```diff
@@ -7,7 +7,7 @@ import {
 } from "../constants";
 import { fetchError, upsertError, deleteError } from "./errors";
 import { actionFactory, validatedApiDispatcher } from "./utils";
-import { apiGET, apiPOST } from "../../libs/apiUtils";
+import { apiGET, apiPOST } from "../../libs/apis";
```

Comment:
I think we should make it `api-utils`. Since there is an `api` directory, it can get really confusing if we have multiple files named `api`.